src/patternfly/sass-utilities/scss-variables.scss

```diff
@@ -198,6 +198,17 @@ $pf-global--arrow--width-lg: pf-font-prem(25px);
 $pf-global--target-size--MinWidth: 44px !default;
 $pf-global--target-size--MinHeight: 44px !default;
 
+// Images
```

Comment:
These are not global variables, neither the syntax is correct.
They belong to the component file.

Reply:
Ok. What should the syntax be?